app/js/user.js

```diff
@@ -116,14 +116,14 @@
     * If the local user data is older than 15 minutes, get a refereshed copy
     * of the user details from the server and return the new data.
     */
-    get_user: function() {
+    get_user: function(force_update) {
```

Comment:
Booleans are very hard to read at call sites. I'd rather create a `User.mark_dirty()` method that would set `user.last_sync` as 0. I think that makes it more clear that we are going to do an HTTP request at call sites.